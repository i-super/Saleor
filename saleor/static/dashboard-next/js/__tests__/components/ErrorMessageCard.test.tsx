import * as React from "react";
import { MemoryRouter } from "react-router-dom";
import * as renderer from "react-test-renderer";

import ErrorMessageCard from "../../components/cards/ErrorMessageCard";
import errorMessageFixture from "./fixtures/errorMessage";

describe("<ProductChildElement />", () => {
  it("renders properly", () => {
    const component = renderer.create(
      <ErrorMessageCard message={errorMessageFixture} />
    );
    expect(component).toMatchSnapshot();
  });
});
