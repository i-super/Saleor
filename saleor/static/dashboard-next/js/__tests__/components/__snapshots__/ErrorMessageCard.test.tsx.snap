// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductChildElement /> renders properly 1`] = `
<div
  className="MuiPaper-root-1 MuiPaper-elevation2-5 MuiPaper-rounded-2"
>
  <div
    className="MuiCardContent-root-28"
  >
    <h2
      className="MuiTypography-root-29 MuiTypography-headline-34"
    >
      Error
    </h2>
    <p
      className="MuiTypography-root-29 MuiTypography-body1-38"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam eget pulvinar purus. Etiam nisl orci, maximus dictum elit vitae, tincidunt scelerisque lacus. Nam sed sem nec arcu gravida facilisis a a nisl. Donec ac orci mollis, porta ligula et, facilisis ipsum. Curabitur ut lacus nisi. Sed dui sapien, lobortis in quam at, vestibulum eleifend dolor.
    </p>
  </div>
</div>
`;
